src/doomtype.h

```diff
@@ -90,6 +91,18 @@ typedef byte lighttable_t;
 #define inline __inline
 #endif
 
+#if defined(__clang__)
```

Comment:
Could you indent this as seen in the other defines?

Reply:
No because gcc vector extension doesn't support xyzw components.
The only way to support it is to switch to array access operator[] for the existing struct vector types.